<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WTM4K Stats</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitpushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showResultspushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PreFixOutputFilePushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string extracomment="Add a name to the start of the csv file"/>
    </property>
    <property name="whatsThis">
     <string extracomment="Add name to start of output file"/>
    </property>
    <property name="text">
     <string>Pre-fix csv name</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>128</width>
      <height>79</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="FCSRadioButton">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string extracomment="in-fcs-errors">Generate in-fcs-errors csv</string>
       </property>
       <property name="text">
        <string>in-fcs-errors</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="InOctetsRadioButton">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>in-octets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GenCSVpushButton">
       <property name="text">
        <string>Generate CSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open_File"/>
    <addaction name="action_Close_File"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_File">
   <property name="text">
    <string>&amp;Open File</string>
   </property>
  </action>
  <action name="action_Close_File">
   <property name="text">
    <string>&amp;Close File</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
